docs/device_api.md

```diff
@@ -397,7 +443,7 @@ In the case of complex commands requiring parameters, the `commandValue` could b
 weatherStation167@ping|param1:1|param2:2
 ```
 
-This example will tell the Weather Station 167 to reply to a ping message with the provided params. Note that `=` cannot be used instead of `:` given that `=` is [a forbidden character for Context Broker](https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html), so the update at CB triggering the command would be never progressed.
+This example will tell the Weather Station 167 to reply to a ping message with the provided params. Note that `=` cannot be used instead of `:` given that `=` is [a forbidden character for Context Broker] (https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html), so the update at CB triggering the command would be never progressed.
```

Comment:
`]` and `(` must be together, in orde the markdown link to work (I guess)

Reply:
OK. To be corrected in the next pull request.